cmd/cinder-csi-plugin/main.go

```diff
@@ -100,6 +109,20 @@ func main() {
 
 func handle() {
 
+	metricsManager := metrics.NewCSIMetricsManager("" /* driverName */)
+
+	mux := http.NewServeMux()
```

Comment:
Why not putting the above 2 lines inside the following `if` section?

Reply:
you are right, updated